code/espurna/scheduler.ino

```diff
@@ -208,19 +239,36 @@ void _schCheck() {
 
     }
 
+    if (daybefore >= 0 && daybefore < 7 && minimum_restore_time == -1440 && saved_action == -1) {
+      _schCheck(relay, ++daybefore);
+      return;
+    }
+
+    if (minimum_restore_time != -1440 && saved_action != -1 && saved_sch != -1) {
+        _schAction(saved_sch, saved_action, relay);
+    }
+
 }
 
 void _schLoop() {
 
     // Check time has been sync'ed
     if (!ntpSynced()) return;
 
+    if (_sch_restore == 0) {
+        for (int i = 0; i < _relays.size(); i++){
+            if (getSetting("relayLastschedule", i, SCHEDULER_RESTORE_LAST_SCHEDULE).toInt() == 1)
```

Comment:
Keep capitalization as `relayLastSchedule`?
Maybe even `relayLastSch`?

Reply:
Replaced relayLastSchedule to relayLastSch as you mentioned